src/renderer/components/settings-general-appearance.tsx

```diff
@@ -193,13 +226,36 @@ export class AppearanceSettings extends React.Component<
             </a>. The easiest way to get started is to clone one of the two existing
             themes and to add your own colors.
           </p>
+          <p>
+            Additionally, if you wish to import an editor theme from VSCode, just pick your JSON file and Fiddle will do the rest.
+          </p>
           <Button
             onClick={this.createNewThemeFromCurrent}
             text='Create theme from current selection'
             icon='duplicate'
           />
+          <Button
+            onClick={this.createNewThemeFromMonaco}
+            text='Add a VSCode Editor theme'
+            icon='duplicate'
+          />
         </Callout>
       </div>
     );
   }
+
+  private async promptForTheme(currWindow: Electron.BrowserWindow, defaultTheme: LoadedFiddleTheme) {
+    const filePicked = await remote.dialog.showOpenDialog(currWindow, {
```

Comment:
Sorry I'm not sure I understand, could you elaborate more on this? I do need the current window in order to call `showOpenDialog` since it is a required parameter.

Reply:
![image](https://user-images.githubusercontent.com/16010076/62961962-b3ce9780-bdb2-11e9-8cf8-4f0b34588d59.png)
https://electronjs.org/docs/api/dialog

Reply:
Ahhh VSCode did me dirty, 😄